librisxl-tools/scripts/crunch-lddb-lines.py

```diff
@@ -0,0 +1,120 @@
+"""
```

Comment:
I'm divided when it comes to adding this script. On the one hand, it might be good as a reference at some point. On the other, if this isn't how we want to do things in the future, it might be better to skip this for now and move on to developing the One True Way(tm).

What are your thoughts?

Reply:
I was equally divided. I think removing it is good since the good parts are easy to redo, the other parts shouldn't be redone. But I'd like to make a note about "lddb-processing" out of a bunch of the "one-off" scripts in here. We could remove this and assemble a separate PR for that note.